packages/ui/Marquee/MarqueeBase.js

```diff
@@ -203,6 +214,12 @@ const MarqueeBase = kind({
 					onTransitionEnd={onMarqueeComplete}
 				>
 					{children}
+					{duplicate ? (
+						<React.Fragment>
+							<div style={{display: 'inline-block', width: padding}} />
```

Comment:
Do we need units on this if it'll be non-zero?

Reply:
Assumed to be pixels by React when its numeric.